physionet-django/environment/views.py

```diff
@@ -112,7 +137,7 @@ def create_research_environment(request, project_slug, project_version):
                 environment_type=form.cleaned_data["environment_type"],
                 persistent_disk=form.cleaned_data.get("persistent_disk"),
             )
-            return redirect("research_environments")
+            return redirect("workspace_setup")
```

Comment:
This is a mistake, no?
```suggestion
            return redirect("research_environments")
```

Reply:
Why redirect there if it's already set up anyway